hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatMap.tsx

```diff
@@ -0,0 +1,342 @@
+      maxSize = Math.max(maxSize, obj.size);
+    }
+    return [minSize, maxSize];
+  };
+    
+  updateSize = (obj: any) => {
+      //Normalize Size so other blocks also get visualized if size is large in bytes minimize and if size is too small make it big
+    if (obj.hasOwnProperty('size') && obj.hasOwnProperty('color')) {
+      let newSize = this.normalize(minSize, maxSize, obj.size);
+      obj['normalizedSize'] = newSize;
+    }
+    if (obj.hasOwnProperty('children')) {
+      obj.children.forEach((child: any) => this.updateSize(child));
+    }
+    return obj;
+  };
+
+  normalize = (min: number, max: number, size: number) => {
+    //Normaized Size using Deviation and mid Point
+    var mean = (max + min) / 2;
+    var highMean = (max+mean)/2;
+    var lowMean1 = (min+mean)/2;
+    var lowMean2 = (lowMean1 + min) / 2;
+      
+    if(size>highMean){
+    var newsize= highMean+(size*0.1);
+    return(newsize);
+    }
+    // lowmean2= 100 value=10, diff=
+    // min= 10 ,max=100, mean=55, lowmean1=32.5,lowmean2=22, value= 15, diff=7, diff/2=3.5, newsize= 22-3.5=18.5
+    if(size<lowMean2){
+    var diff = (lowMean2-size)/2;
+    var newSize= lowMean2-diff;
+    return(newSize);
+    }
+    return size;
+  }
+
+  render() {
+    const { treeResponse, isLoading, inputPath, date} = this.state;
+    const menuCalender = (
+      <Menu
+        defaultSelectedKeys={[date]}
+        onClick={e => this.handleCalenderChange(e)}>
+        <Menu.Item key='24H'>
+        24 Hour
+        </Menu.Item>
+        <Menu.Item key='7D'>
+        7 Days
+        </Menu.Item>
+        <Menu.Item key='90D'>
+        90 Days
+        </Menu.Item>
+        <Menu.SubMenu  title="Custom Select Last 90 Days">
+            <Menu.Item> 
+              <DatePicker
+                format="YYYY-MM-DD"
+                onChange={this.onChange}
+                disabledDate={this.disabledDate}
+              />
+          </Menu.Item>
+        </Menu.SubMenu>
+      </Menu>
+    );
+    
+    const entityTypeMenu = (
+      <Menu
+        defaultSelectedKeys={[this.state.entityType]}
+        onClick={e => this.handleMenuChange(e)}>
+          <Menu.Item key='volume'>
+            Volume
+          </Menu.Item>
+          <Menu.Item key='bucket'>
+            Bucket
+          </Menu.Item>
+          <Menu.Item key='key'>
+            Key
+          </Menu.Item>
+      </Menu>
+    );
+
+    return (
+      <div className='heatmap-container'>
+        <div className='page-header'>
+          Tree Map for Entities
+        </div>
+        <div className='content-div'>
+          { isLoading ? <span><Icon type='loading'/> Loading...</span> : (
+            <div>
+              {(Object.keys(treeResponse).length > 0 && (treeResponse.minAccessCount > 0 || treeResponse.maxAccessCount > 0)) ?
+                (treeResponse.size === 0)? <div style={{ height: 800 }} className='heatmapinformation'><br />Failed to Load Heatmap.{' '}<br/></div>
```

Comment:
Shall we move the `style={{ height: 800}}` to the `heatmapinformation` class in the `heatmap.less`?